lib/streamlit/elements/lib/built_in_chart_utils.py

```diff
@@ -675,6 +686,8 @@ def _get_opacity_encoding(
 ) -> alt.OpacityValue | None:
     import altair as alt
 
+    # Since opacity set to 0.7 for all area charts previously, avoid
```

Comment:
what do you mean with `previously`? the only other line in the function before this is an `import` statement. Probably you mean the code that called this function, but I don't think its a good idea to refer in the comment to a caller as this could change in the future or other functions might call this one here.